### ISSUE TITLE
Replace PNG image with an SVG

### ISSUE THREAD

**Issue by dartiss** [dartiss/draft-list#37]
Will also remove the option to specify your own image.

**Comment by dartiss** [comment]
I've tested this code and it works...

```
/**
 * Add Dashicons to front-end
 *
 * Will enqueue Dashicons for front-end use.
 */
function adl_load_dashicons_front_end() {
	wp_enqueue_style( 'dashicons' );
}

add_action( 'wp_enqueue_scripts', 'adl_load_dashicons_front_end' );
```

```
$icon_url = '<span class="dashicons dashicons-clock">';
```

**Comment by dartiss** [comment]
Implemented in #68
